apps/integration/src/app/integrations/field-heights/field-heights.component.html

```diff
@@ -5,24 +5,15 @@
   (ngSubmit)="onSubmit()"
 >
   <div class="sky-form-group">
-    <sky-input-box stacked="true">
-      <label class="sky-control-label" [attr.for]="favoriteText.id">
-        Text field
-      </label>
-      <input
-        class="sky-form-control"
-        type="text"
-        formControlName="favoriteText"
-        skyId
-        #favoriteText="skyId"
-      />
+    <sky-input-box stacked="true" labelText="Text field" hintText="Enter text.">
```

Comment:
Should we include both "hard" and "easy" mode versions to make sure we account for both?

Reply:
We chatted offline. I’m going to merge this to unblock other PRs and then do a PR for hard mode.